src/MessagingInput/MessagingInput.less

```diff
@@ -1,5 +1,9 @@
 @import (reference) "../less/index";
 
+.MessagingInput--Container {
+  position: relative;
```

Comment:
similar to my other question- just for my learning when you have time - i havent quite figured out when to use `position: relative`. What does it add here?

Reply:
When an el A has `position: absolute`, it looks for the nearest parent B that has `position: relative` - el A is absolute relative to that el B.

There's more to it than that, but that's why it's here. 🙂

Reply:
Oh, interesting! Thank you, that makes sense